src/Toolkit/A.php

```diff
@@ -726,4 +726,53 @@ public static function wrap($array = null): array
             return $array;
         }
     }
+
+    /**
+     * Filter the array using the given callback
+     * using both value and key
+     *
+     * @param array $array
+     * @param callable $callback
+     * @return array
+     */
+    public static function where(array $array, callable $callback): array
```

Comment:
I think calling this `filter` would make more sense and be more in line to the PHP name as well as similar to our collection `->filter($callback)` methods.

Reply:
This one is actually nod to `Arr::where` from Laravel, but they also support other stuff (dot syntax, etc). I'll switch it to filter